NodeReact/Utils/PooledStream.cs

```diff
@@ -15,13 +15,16 @@ static PooledStream()
         var largeBufferMultiple = 1024 * 1024;
         var maximumBufferSize = 128 * 1024 * 1024;
         
-        _manager = new RecyclableMemoryStreamManager(blockSize, largeBufferMultiple, maximumBufferSize)
+        _manager = new RecyclableMemoryStreamManager(new RecyclableMemoryStreamManager.Options()
         {
+            BlockSize = blockSize,
+            LargeBufferMultiple = largeBufferMultiple,
+            MaximumBufferSize = maximumBufferSize,
             GenerateCallStacks = true,
```

Comment:
I played around with my own fork inspired by yours and got an error during hot reload.

```
Unable to cast object of type 'System.Attribute[]' to type 'System.Runtime.CompilerServices.StateMachineAttribute[]'.'
```

Turns out it was `GenerateCallStacks = true,` that did not play nice with hot reload, setting this to false solved the problem for me.